metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -37,6 +37,12 @@
         <display-name>Metron apps indexed HDFS dir</display-name>
     </property>
     <property>
```

Comment:
I think if we move the error type into the source.type field it will solve this.  I believe the source type is added to the end of the HDFS path.